lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -88,9 +96,23 @@ class IntegrationTestsManager {
         useSystemFlutter: _useSystemFlutter);
   }
 
-  void _runDriver() async {
+  /// Start the Chrome Driver on LUCI.
+  ///
+  /// Driver should already exist on LUCI as a CIPD package. No need to install.
+  ///
+  /// Throw an error if directory does not exists.
+  void runDriverOnLUCI() {
+    if (!_browserDriverDir.existsSync()) {
+      throw StateError('Failed to locate Chrome driver on LUCI on path:'
+          '${_browserDriverDir.path}');
+    }
+    print('INFO: Starting the driver on path: ${_browserDriverDir.path}');
+    _runDriver(_browserDriverDir.path);
+  }
+
+  void _runDriver(String workingDirectory) async {
     startProcess('./chromedriver/chromedriver', ['--port=4444'],
-        workingDirectory: io.Directory.current.path);
+        workingDirectory: workingDirectory);
```

Comment:
This is the implementation of the method. I thought it would `await` for the process and will add it to the cleanup list.

We are using this method since March: https://github.com/flutter/engine/pull/17190/files#diff-9e0e5a53a06c2f6ab3e0904639468ab6R110

```
final io.Process process = await io.Process.start(
    executable,
    arguments,
    workingDirectory: workingDirectory,
    // Running the process in a system shell for Windows. Otherwise
    // the process is not able to get Dart from path.
    runInShell: io.Platform.isWindows,
    mode: io.ProcessStartMode.inheritStdio,
  );
  processesToCleanUp.add(process);
```

Please let me know if I'm missing something.

Reply:
I'm not talking about clean-up. I'm talking about start-up race condition. The `_startDriver` method returns before the process is started because it swallows the `Future` returned by `startProcess`.

Reply:
I sent a conversation offline. Let's discuss if there is leak in the process. I can send another fix since the methods in utils.dart are used by other places. I can fix them all in one PR.

Reply:
As discussed the change was running without the Future (await'in on void) due to our analyze options: https://github.com/flutter/engine/blob/master/lib/web_ui/analysis_options.yaml#L30